docs/concepts/scale-up.md

```diff
@@ -0,0 +1,44 @@
+## Scaling-up the single-node to multi-node etcd cluster deployed by etcd-druid
+
+To mark the cluster for scale-up from single node to multi-node etcd just patch the etcd custom resource's `.spec.replicas` from `1` to `3`(say).
+
+### Challenges for scale-up
+1. Etcd cluster with single replica don't have any peer, so no peer communication is required hence peerUrl may or may not be TLS enabled but while scaling up from single node etcd to multi-node etcd cluster there will be a requirement to have a peer communication between each members of etcd cluster. Peer communication is required for various reasons like to get in a sync with each other cluster state, data and to perform leader election or any cluster wide operation like to remove/add a member etc. Hence in multi-node etcd cluster we need to have TLS enable peer url for peer communication.
+2. Providing the correct configuration to start a new etcd members as it is different from boostrapping a cluster since this new etcd members will join the existing cluster.
```

Comment:
```suggestion
2. Providing the correct configuration to start a new etcd members as it is different from boostrapping a cluster since this new etcd members will join an existing cluster.
```